model/Core/Classes/PackageVerificationCode.md

```diff
@@ -18,19 +18,21 @@ This identifier enables a recipient to determine if any file in the original pac
 
 Algorithm:
 
+    hashValue = 0
     templist = empty list of strings
 
-    for all files in the package
-        if file is an "excludes" file
+    for all files in the package {
```

Comment:
Using indentation for grouping works fine, why introduce brackets in this pseudo-language?

Reply:
Style preference.     Using C style comments, so used C style bracketing to be explicit about multiple statements, esp if indenting/formating gets mucked up.